include_core/ute_dataformat.h

```diff
@@ -116,6 +116,7 @@ typedef enum {
 	UT_PIV,
 	UT_TREX,
 	UT_OPTERON,
+	UT_RISCV,
```

Comment:
Maybe I'm missing something but this looks like a definition of u-architecture or variant, so maybe something like `UT_RV64G` (instead of `UT_RISCV`) is more appropriate? 
(All the RISC-V code so far kind of expects RISC-V 64bit with IMAFD extensions)

Reply:
Agreed and updated to `UT_RV64G` as suggested above.